python/src/RandomVectorMetropolisHastings_doc.i.in

```diff
@@ -42,22 +42,22 @@ We can also link the parameters of the random vector to the state of the chain,
 here the parameters of the random vector being the ones of the underlying distribution:
 
 >>> slf = ot.SymbolicFunction(['x'], ['0.0', '0.1'])
->>> sampler = ot.RandomVectorMetropolisHastings(source, initialState, [0], slf)"
+>>> sampler = ot.RandomVectorMetropolisHastings(randomVector, initialState, [0], slf)"
 
```

Comment:
I think the RandomVectorMetropolisHastings API doc page should mention that the class is designed to work with the `Gibbs` class and link to the "Gibbs sampling of the posterior distribution" example.

Reply:
It's simpler to use "See Also" for the relevant classes